### ISSUE TITLE
Mouse events not fired in IE11 running fullscreen when canvas is in iframe

### ISSUE THREAD

**Issue by Excaa** [CreateJS/EaselJS#610]
Stage does not react to mouse events in internet explorer version 11 (other ie versions might be affected also) when the canvas is inside an iframe and the canvas element is toggled to fullscreen. 

The bug can be seen in action in here:
https://dl.dropboxusercontent.com/u/8932415/createjs_ie11_fs/iframe.html

The iframes content is here:
https://dl.dropboxusercontent.com/u/8932415/createjs_ie11_fs/index.html

When the html file is run directly it works. Click, mousedown and mouseup events for the canvas-element are logged to console and they trigger correctly in fs+iframe.

**Comment by lyubo-slavilov** [comment]
Yes - same here. When tried to use our game (embeded as facebook app) in fullscreen in IE11, the Stage went ridicilous.
Events are fired, but the values of the mouse position are abnormal:

    [MouseEvent (type=mousedown stageX=82 stageY=129)]
    {
          [functions]: ,
      __proto__: { },
      bubbles: true,
      cancelable: false,
      currentTarget: { },
      defaultPrevented: false,
      eventPhase: 2,
      immediatePropagationStopped: false,
      isTouch: false,
      localX: -111460.74329112736,     <-----------
      localY: 61108.98945918314,        <-----------
      nativeEvent: { },
      pointerID: -1,
      primary: true,
      propagationStopped: false,
      rawX: -106379.5868698621,     <-----------
      rawY: 57999.998975683164,     <-----------
      removed: false,
      stageX: 82,
      stageY: 129,
      target: { },
      timeStamp: 1432897798974,
      type: "mousedown"
     }

All other browsers with FullscreenAPI support seems fine
